A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_CRZ.js

```diff
@@ -67,67 +86,102 @@ var A320_Neo_LowerECAM_CRZ;
             }
             
             // Fuel
-            var unitFactor = SimVar.GetSimVarValue("FUEL WEIGHT PER GALLON", "lbs");
-            if (BaseAirliners.unitIsMetric(Aircraft.A320_NEO)) {
-                unitFactor = SimVar.GetSimVarValue("FUEL WEIGHT PER GALLON", "kilogram");
-            }
-            let leftConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:1", "gallon") * unitFactor * 0.001;
-            let rightConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:2", "gallon") * unitFactor * 0.001;
+            const leftConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:1", "gallon") * this.unitFactor * 0.001;
+            const rightConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:2", "gallon") * this.unitFactor * 0.001;
+
+            const leftConsumptionShown  = fastToFixed(leftConsumption - (leftConsumption % 10), 0);
+            const rightConsumptionShown = fastToFixed(rightConsumption - (rightConsumption % 10), 0);
 
-            this.FuelUsedTotal.textContent = Math.round(leftConsumption)+Math.round(rightConsumption);
-            this.FuelUsedLeft.textContent = Math.round(leftConsumption);
-            this.FuelUsedRight.textContent = Math.round(rightConsumption);
+            if ((leftConsumptionShown != this.LeftConsumptionDisplayed) || (rightConsumptionShown != this.RightConsumptionDisplayed)) {
```

Comment:
I would change the names, "shown" and "displayed" are confusingly similar

Reply:
Asobo's pattern for value diffing uses the parameter pattern: `last<VariableName>`. Might be best to follow the same pattern so it's consistent

Reply:
Okay, I will do that